docs/public/homepage.scss

```diff
@@ -0,0 +1,303 @@
+@font-face {
```

Comment:
You can move this file to `.vitepress/theme` and import it in `.vitepress/theme/index.ts`, it's the main JS entry

Reply:
Perfect, that did it - good to go